.eslintrc.js

```diff
@@ -0,0 +1,69 @@
+// @ts-check
+
+/** @type { import('eslint').Linter.Config } */
+const config = {
+  extends: [
+    'eslint:recommended', // https://eslint.org/docs/rules/
+    'plugin:react/recommended', // https://github.com/jsx-eslint/eslint-plugin-react#list-of-supported-rules
+    'plugin:@typescript-eslint/recommended', // https://typescript-eslint.io/rules/
+    'react-app', // https://github.com/facebook/create-react-app/blob/main/packages/eslint-config-react-app/index.js
+    'plugin:prettier/recommended', // See .prettierrc
+  ],
+  parser: '@typescript-eslint/parser',
+
+  settings: {
+    // Warn about <a target="_blank" rel="noopener"> in components other than "a"
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md#custom-link-components
+    linkComponents: ['Chip', 'Link', 'SocialLink', 'StyledLink'],
+  },
+  rules: {
+    'no-restricted-imports': [
+      'error',
+      {
+        paths: [
+          {
+            name: '@mui/material',
+            message: "Use '@mui/material/*' instead.",
+          },
+        ],
+        patterns: [
+          {
+            group: ['@mui/system'],
+            message: "Use '@mui/material/styles' instead.",
+          },
+          {
+            group: ['oasis-index/generated/api'],
```

Comment:
This is wrong

Reply:
`**/oasis-indexer/generated/api` should work

Reply:
Right.